enterprise/server/registry/registry.go

```diff
@@ -813,7 +928,13 @@ func main() {
 	flag.Parse()
 
 	if err := flagutil.PopulateFlagsFromFile(config.Path()); err != nil {
-		log.Fatalf("Error loading config from file: %s", err)
+		fmt.Printf("Error loading config from file: %s", err)
+		os.Exit(1)
```

Comment:
I see you changed this from log.Fatalf -- what's the advantage of doing it this way?